GitHawkRoutesTests/GitHawkRoutesTests.swift

```diff
@@ -11,6 +11,15 @@ import XCTest
 
 class GitHawkRoutesTests: XCTestCase {
 
-    
+    func testRepo() {
+        let repo = RepoRoute(
+            owner: "GitHawkApp",
+            repo: "GitHawk",
+            branch: "master"
+        )
+        let url = URL.from(githawk: repo)
+        
+        XCTAssertNotNil(url)
```

Comment:
Can you assert the full URL? Should be:
```
freetime://com.githawk.repo?owner=GitHawkApp&repo=GitHawk&branch=master
```

Reply:
Love that you're adding tests btw!!

Reply:
Thank you. Having the specific target for that route also made it clear how to fix it.

My only problem is that ````components.queryItems = route.encoded.map(URLQueryItem.init)```` doesn't order the parameters the same way in each test run, such that it would need a pretty complicated unit test as equality matching doesn't work.

````
XCTAssertEqual failed: 
("Optional(freetime://com.githawk.repo?owner=GitHawkApp&branch=master&repo=GitHawk)") is not equal to 
("Optional(freetime://com.githawk.repo?owner=GitHawkApp&repo=GitHawk&branch=master)")
````

What would you prefer?
- a simple non-specific test
- a specific complicated test that parses the url's testing on the component level
- deterministic url-generation with fixed order of parameters

Reply:
We could create another `URLComponents` object and assert on that?